pkgs/development/libraries/graphene/default.nix

```diff
@@ -59,6 +59,8 @@ stdenv.mkDerivation rec {
     gobject-introspection
     python3
     makeWrapper
+  ] ++ lib.optionals (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
```

Comment:
We should really push this in a treewide change but thats a topic for another PR.

Reply:
`gtk-doc` always needs this when cross-compiling so it could propagate it